src/future/server/shutdown.rs

```diff
@@ -81,28 +81,28 @@ impl Watcher {
 
     fn poll_shutdown_requests(&mut self) -> Poll<Option<()>, ()> {
         Ok(Async::Ready(match try_ready!(self.shutdown_rx.poll()) {
-            Some(tx) => {
-                debug!("Received shutdown request.");
-                self.shutdown = Some(tx);
-                Some(())
-            }
-            None => None,
-        }))
+                            Some(tx) => {
+            debug!("Received shutdown request.");
```

Comment:
The heuristic for "this block is sufficiently complex to be block indented" kicks in I guess. I think this is a known bug sort of though.